docs/deployment/one-off-processes.md

```diff
@@ -4,8 +4,9 @@ Sometimes you need to either inspect running containers or run a one-off command
 
 ## Run a command in an app environment
 
-```
-run <app> <cmd>                                  Run a command in the environment of an application
+```shell
+# Run a command in the environment of an application
+run <app> <cmd>
```

Comment:
Do not reformat these codeblocks please. Will mean updating the help message requires more work.